src/components/Menu/styles/basic/StyledItemLink.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+import Link from 'components/Link'
+
+const ItemLink = styled(Link)`
+  height: 100%;
+  padding: 0 ${({ theme }) => theme.components.menu.basic.link.spaces.paddingWidth};
```

Comment:
You should deconstruct `theme.components.menu.basic.link` because it's way too long and you use it multiple times

Reply:
deconstruct `theme.components.menu.basic.link` is not a consistant with current. I think a provider/helper should be created to prevent a long path to get key of object. 

maybe this lib has something like this : https://github.com/diegohaz/styled-tools

Reply:
Comment moved to issue #56